### DISCUSSION TITLE
Значение в циклической по Селекту, с данными по link изнутри

### DISCUSSION THREAD

**Discussion by Alexmeb** [totumonline/totum-ru-issues-and-discussions#184] [Как это сделать?] [answered]
В циклической таблице поле селект, при выборе определенного значения нужно заменить значение в другом столбце, но это значение можно менять только внутри цикла и оно связано через link внутрь.  Как реализовать чтобы не зациклить таблицу?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
`set` внутрь цикла. В строке таблицы циклов можно не указывать параметр `cycle:` так-как он будет подставлен автоматически равным `id` строки.

**Reply by Alexmeb** [reply]
Разобрался. Добавил recalc по коду действия внутри цикла в циклической, всё работает :)
